x/magpie/internal/keeper/keeper_test.go

```diff
@@ -1,12 +1,51 @@
 package keeper_test
 
 import (
+	"fmt"
+	"math"
 	"testing"
 
 	"github.com/desmos-labs/desmos/x/magpie/internal/types"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestKeeper_SetDefaultSessionLength(t *testing.T) {
+	tests := []int64{-1, 0, 1, math.MaxInt32}
+
+	for _, length := range tests {
+		length := length
+
+		t.Run(fmt.Sprintf("Default session length: %d", length), func(t *testing.T) {
+			ctx, k := SetupTestInput()
+			k.SetDefaultSessionLength(ctx, length)
+
+			var stored int64
+			store := ctx.KVStore(k.StoreKey)
+			k.Cdc.MustUnmarshalBinaryBare(store.Get([]byte(types.SessionLengthKey)), &stored)
+			assert.Equal(t, length, stored)
+		})
+	}
+}
+
+func TestKeeper_GetDefaultSessionLength(t *testing.T) {
+	tests := []int64{-1, -2, 0, 1, 2}
```

Comment:
why testing invalid lengths such as -1, -2 and 0?

Reply:
Removed such cases, it's OK to not test them.